source/pages/solar-position.ipynb

```diff
@@ -12,12 +12,12 @@
     "1) The solar position algorithm used by the API is equivalent to pvlib's implementation of the SPA to within the precision of the values returned from the API (2 digits after the decimal).\n",
```

Comment:
<p>Is there a space missing between "the" and "true"?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/AssessingSolar/unofficial-psm3-userguide/pull/5/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Reply:
I'm not following.  Maybe I'm missing something, but I don't see anything matching this description in this notebook.  (I'm looking at this on GitHub, not on ReviewNB)

Reply:
Could just be me incorrectly viewing the notebook, but I see this using ReviewNB:
![image](https://user-images.githubusercontent.com/39184289/210024323-900d2280-e034-441c-845e-78f6b112d071.png)

It seems just to be an artifact, as it's not there when I look at the notebook standalone.